bluecode-pay/src/test/java/com/example/bluecodepay/bluecode/transformer/BluecodePaymentTransformerTest.java

```diff
@@ -31,7 +29,7 @@ void fromDto() {
     }
 
     @Test
-    void transformToPaymentResponseDto() {
+    void shouldTransformToPaymentResponseDto() {
         BluecodeResponseDto paymentDto = BLUECODE_RESPONSE_DTO.toBuilder()
```

Comment:
Could we place these objects inside TestConstants? In case if we don't need some fields we can use toBuilder to remove them